core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java

```diff
@@ -223,12 +227,18 @@ public void testWrap() throws Exception {
 		String actual = prr.getResult().asString();
 
 		TestAssertions.assertEqualsIgnoreCRLF(expected, actual);
+
+		List<String> warnings = getConfigurationWarnings().getWarnings();
+		assertEquals(1, warnings.size());
+		assertTrue(warnings.get(0).contains("should NOT be used to wrap a message in an InputWrapper"));
 	}
 
 	@Test
 	public void testWrapSoapVersionSoap12() throws Exception {
 		pipe.setOutputNamespace(TARGET_NAMESPACE);
 		pipe.setSoapVersion(SoapVersion.SOAP12);
+		pipe.setName(PipeLine.INPUT_WRAPPER_NAME);
+		pipeline.addPipe(pipe);
```

Comment:
De setup doet dit al

Reply:
Nee helaas @nielsm5, dan linkt hij niet meer deze pipe aan de pipeline, omdat de naam niet meer klopt. Ik moest dit doen om hem weer vindbaar te krijgen.

Reply:
Ah, okido! Ik neem aan dat het scenario fout gaat zonder deze change. Dan is het verder prima!

Reply:
Klopt. Thanks!